editor/editor_help.cpp

```diff
@@ -813,34 +813,24 @@ void EditorHelp::_update_doc() {
 	}
 
 	// Descendents
```

Comment:
```suggestion
	// Descendants
```

Unrelated, but I know you wouldn't miss an opportunity here :)